src/Microsoft.AspNetCore.Mvc.TagHelpers/FormTagHelper.cs

```diff
@@ -240,6 +240,7 @@ public override void Process(TagHelperContext context, TagHelperOutput output)
                 var antiforgeryTag = Generator.GenerateAntiforgery(ViewContext);
```

Comment:
Why not update the `HasAntiforgeryToken` property in the `GenerateAntiforgery()` method? No longer have any cases where it's _not_ set.